Star Trek Adventurers 2d20/sheet.json

```diff
@@ -0,0 +1,7 @@
+{
+	"html": "Star Trek Adventures 2d20.html",
+	"css": "Star Trek Adventures 2d20.css",
+	"authors": "David Collier",
+	"preview": "Star Trek Adventures 2d20.PNG",
```

Comment:
Dunno if its really required or not, but do you need to add your roll20 user id in here?